opencti-platform/opencti-graphql/src/modules/internal/csvMapper/csvMapper-domain.ts

```diff
@@ -48,6 +50,20 @@ export const fieldPatchCsvMapper = async (context: AuthContext, user: AuthUser,
 };
 
 export const deleteCsvMapper = async (context: AuthContext, user: AuthUser, csvMapperId: string) => {
+  // detect if ingesters are using this mapper
+  const opts = {
+    filters: {
+      mode: FilterMode.Or,
+      filterGroups: [],
+      filters: [{ key: ['csv_mapper_id'], values: [csvMapperId] }]
+    }
+  };
+  const ingesters = await listAllEntities<BasicStoreEntityIngestionCsv>(context, user, [ENTITY_TYPE_INGESTION_CSV], opts);
+  // prevent deletion if an ingester uses the mapper
+  if (ingesters.length > 0) {
+    throw FunctionalError('Cannot delete this CSV Mapper: it is used by one or more IngestionCsv ingester(s)', { id: csvMapperId });
```

Comment:
maybe "CSV feed" instead of "IngestionCsv" ? since it's the term we use in the platform